src/constants.tsx

```diff
@@ -4,14 +4,19 @@ import systemPropTypes from '@styled-system/prop-types'
 import themeGet from '@styled-system/theme-get'
 import theme from './theme'
 
+declare module 'styled-system' {
```

Comment:
What is this used for?

Reply:
The symbols `TYPOGRAPHY`, `COMMON`, and `BORDER` are all of type `styleFn` from the `'styled-system'` package, and this `styleFn` type doesn't have a member called `propTypes`. This declaration augments the `styleFn` type to include a `propTypes` member that can be of `any` type.